Add clarification around statusList and revocation support for Domain Linkage Credentials (#54)
decentralized-identity/jwt-vc-presentation-profile: spec/spec.md

To strengthen trust between the Verifier/RP and End-user, a Verifier/RP's DID must be bound to its website. This proves the Verifier/RP controls both the DID and the origin and allows the End-user to verify this relationship. To bind an owner of a DID to a controller of a certain origin, Well Known DID Configuration MUST be used as defined in [[ref: Well Known DID]].

> Validation of Domain Linkage Credentials by the wallet MUST follow the steps given in the [[ref: Well Known DID]] specification. Additional checks, e.g. of revocation, are not required by this profile. Since Domain Linkage Credentials are being managed by the Verifier/RP, updating the DID Configuration Resource directly, the usage of a credentialStatus property for revocation in a Domain Linkage Credential typically is of little use. Expiration of the Domain Linkage Credential does need to be taken into account.

Reviewer: LGTM. I wonder if we could make the last sentence a little clearer?
```suggestion
Validation of Domain Linkage Credentials by the wallet MUST follow the steps given in the [[ref: Well Known DID]] specification. Additional checks, e.g. of revocation, are not required by this profile. Since Domain Linkage Credentials are being managed by the Verifier/RP, updating the DID Configuration Resource directly, the usage of a credentialStatus property for revocation in a Domain Linkage Credential typically is of little use. To check validity of the Domain Linkage Credential, expiration property can be taken into account.
```

Author: Can suggests optionality, whilst it is required in the well-known did spec.

Maybe just have 'To check validity of the Domain Linkage Credentials, expiration needs to be taken into account`?

Reviewer: `exp` is optional in a Domain Linkage Assertion, right?

> The JWT object SHOULD contain the exp property, and if present, the value MUST be an epoch time that is greater than the current epoch time. If this property is not present, the assertion's future expiry is based on its continued presence in the resource (or its replacement with updated assertion that contains an expiry value).

https://identity.foundation/specs/did-configuration/#validation-of-domain-linkage-assertions

Reviewer: probably also align the language in the profile to the language in the .well-known and change `Domain Linkage Credential` to `Domain Linkage Assertion`?

Author: @Sakurann That looks like an outdated or initial version of the spec. The current spec mentions that expiration date must be present and calls them Domain Linkage Credentials: https://identity.foundation/.well-known/resources/did-configuration/#domain-linkage-credential

Reviewer: Understood. I updated the suggested text above. happy to approve the PR if you incorporate those changes.

Author: Updated with your suggestion. PR should be ready for approval
